Observing-Interface-Setup/environments.rst

```diff
@@ -0,0 +1,174 @@
+- Slack: ncsa-integ-teststand
+
+
+.. _Observing-Interface-Environments-BTS:
+
+Base Test Stand (BTS)
+---------------------
+
+BTS is a simulation environment running at the base facility in La Serena.
+The main difference between the :ref:`NTS <Observing-Interface-Environments-NTS>` and this environment is that some systems will run using hardware simulators.
+
+Hardware simulators are, in general, higher fidelity than those of pure software available at :ref:`NTS <Observing-Interface-Environments-NTS>`.
+For that, BTS will be used mostly for higher fidelity testing, development and debugging.
+
+- Nublado: https://base-lsp.lsst.codes/
+- Chronograf: https://chronograf-base-efd.lsst.codes/
+- LOVE: N/A
+- ArgoCD: https://base-lsp.lsst.codes/argo-cd/
+- Slack: N/A
+
+.. _Observing-Interface-Environments-TTS:
+
+Tucson Test Stand (TTS)
+-----------------------
+
+TTS is being assembled in Tucson and is still not available for use.
+This environment will be similar to that of :ref:`NTS <Observing-Interface-Environments-NTS>` and will be used mainly for development of deployment strategies.
+
+- Nublado: https://tucson-teststand.lsst.codes/
+- Chronograf: https://chronograf-tucson-teststand-efd.lsst.codes/
+- LOVE: N/A
+- ArgoCD: https://tucson-teststand.lsst.codes/argo-cd
+- Slack: N/A
+
+.. _Observing-Interface-Environments-Summit:
+
+Summit
+------
+
+Summit is our main production environment.
+Systems running here will be directly controlling hardware or communicating with components that control actual hardware.
+
+  .. important::
+
+      In the case of the Summit it is required to have personnel present at the site prior to any activity that involves moving hardware.
+      You may also be required to issue a jira ticket in the `summit activity project <https://jira.lsstcorp.org/projects/SUMMIT>`__.
+
+- Nublado: https://summit-lsp.lsst.codes/
+- Chronograf: https://chronograf-summit-efd.lsst.codes/
+- LOVE: http://amor01.cp.lsst.org/
+- ArgoCD: https://summit-lsp.lsst.codes/argo-cd
+- Slack: N/A
+
+.. _Observing-Interface-Getting-Started-Troubleshooting:
+
+Troubleshooting
+===============
+
+.. This section should include troubleshooting information. Information in this section should be strictly related to this procedure.
+
+.. If there is no content for this section, remove the indentation on the following line instead of deleting this sub-section.
+
+If you can not open the links to the environment you intend to work with, make sure you are connected to the LSST-WAP wifi network in one of the designed areas (Tucson, La Serena or Summit facilities) or that you are connected to the NOAO VPN.
+
+If problems persist, you can ask for help in the designed Slack channels or in the com-square channel.
+
+.. _Observing-Interface-Getting-Started-Personnel:
+
+Contact Personnel
+=================
+
+This procedure was last modified |today|.
+
+This procedure was written by |author|. The following are contributors: |contributors|.
```

Comment:
feel free to change this to make it more useful. No need to stick to the template if you don't want to.
you could imagine putting, "for questions involving interface connection problems, ask X. For issues involving LOVE, contact Y" or whatever you wish.